ledger/participant-integration-api/src/main/scala/platform/store/dao/events/EventsTableFlatEvents.scala

```diff
@@ -165,7 +165,7 @@ private[events] object EventsTableFlatEvents {
       filter: FilterRelation,
       pageSize: Int,
   ): SqlSequence[Vector[EventsTable.Entry[Raw.FlatEvent]]] =
-    getFlatTransactionsQueries(sqlFunctions)(
+    getFlatTransactionsQueries(sqlFunctions).apply(
```

Comment:
```suggestion
    getFlatTransactionsQueries(sqlFunctions)(
```

Reply:
good eye! thanks! must be some sawdust. removed

Reply:
Sawdust 😆